sstables/sstables.hh

```diff
@@ -283,11 +283,13 @@ public:
 
     // A reader which doesn't use the index at all. It reads everything from the
     // sstable and it doesn't support skipping.
+    using integrity_check = bool_class<class integrity_check_tag>;
     mutation_reader make_crawling_reader(
             schema_ptr schema,
             reader_permit permit,
             tracing::trace_state_ptr trace_state = {},
-            read_monitor& monitor = default_read_monitor());
+            read_monitor& monitor = default_read_monitor(),
+            integrity_check integrity = integrity_check::no);
```

Comment:
Here we cannot avoid the default, because the above already have defaults. So it is fine here.